notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1294 @@
+{
```

Comment:
<p>Consider explaining that the minimum amount of time that you have to wait for an alert is one hour and it can't be changed to less as of now. I think customers would like it to be sooner but unfortunately, Vertex does not let you configure it to be less than an hour.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>